jinahub/indexers/query/vector/FaissIndexer/__init__.py

```diff
@@ -72,8 +77,54 @@ def __init__(self,
         self.distance = distance
         self.normalize = normalize
         self.nprobe = nprobe
+        self.on_gpu = on_gpu
+
+        self.default_top_k = default_top_k
+        self.traverse_path = traverse_path
+
+        self.logger = get_logger(self)
+
+        dump_path = dump_path or kwargs.get('runtime_args').get('dump_path')
+        if dump_path is not None:
+            self.logger.info('Start building "AnnoyIndexer" from dump data')
+            ids, vecs = import_vectors(dump_path, str(self.runtime_args.pea_id))
+            self._ids = np.array(list(ids))
+            self._ext2int = {v: i for i, v in enumerate(self._ids)}
+            self._vecs = np.array(list(vecs))
+            self.num_dim = self._vecs.shape[1]
+            self.dtype = self._vecs.dtype
+            self.index = self._build_index(self._vecs)
+        else:
+            self.logger.warning(
+                'No data loaded in "AnnoyIndexer". Use .rolling_update() to re-initialize it...'
+            )
+
+    @cached_property
```

Comment:
avoid using `cached_property`. This only made sense when executors where `dumped`. Use a `private` member instead or a simple property

Reply:
done